feat(wallet): display amount in USD on send review screen

vansh2408 opened the pull request:
# Motivation
- Improves user transparency during the review step.
- Aligns with how the balance is displayed in USD.

## Issue Reference
Closes #1999
<!-- Describe the motivation that lead to the PR -->

# Display Amount in USD on Wallet Send Review Screen

## Description
This PR enhances the **wallet send review screen** by displaying the **amount in USD** in addition to ICP.  
Previously, only the balance was shown in USD. Now, both the **balance** and the **amount being sent** are displayed in dollars, improving user clarity.

## Changes
- Updated `SendTokensAmount.svelte`:
  - Added conditional block to render the amount in USD.
  - Used `formatICPToUsd({ icp: token, icpToUsd: $icpToUsd })` to calculate conversion.
- Styled the USD amount with `.usd` class for consistency with balance display.

vansh2408 committed:
feat(wallet): display amount in USD on send review screen

peterpeterparker commented on src/frontend/src/lib/components/tokens/SendTokensAmount.svelte:
```diff
@@ -18,5 +19,16 @@
 		{#if nonNullish(token)}<span>{formatICP(token.toE8s())} <small>ICP</small></span>{/if}
+
+		{#if nonNullish($icpToUsd) && nonNullish(token)}
+			<span class="usd">{formatICPToUsd({ icp: token, icpToUsd: $icpToUsd })}</span>
```
`token` is of type `TokenAmountV2` and `icp:` is expecting a `bigint`, that's why here you need to use `token.toE8s()`.

```
<span class="usd">{formatICPToUsd({ icp: token.toE8S(), icpToUsd: $icpToUsd })}</span>
```

vansh2408 replied:
Resolved this.

peterpeterparker reviewed (commented):
Looks good. Spotted a small mistake if you can adjust it.

Aside from that, not a must, but if you can run `npm run format && npm run lint -- --fix` to make the linter happy, that would be super but, I can take care of it too after merge, I don't see it as a blocker.

vansh2408 committed:
Changed token type from TokenAmountV2 to bigint

peterpeterparker committed:
Update SendTokensAmount.svelte

Fix a small typo. It was on me when I provided the snippet in my previous review.

peterpeterparker committed:
Merge branch 'main' into feature/display-wallet-send-amount-usd

peterpeterparker reviewed (approved):
LGTM, thx!

Happy Hacktoberfest! 🎃🥳
